client/signup/main.jsx

```diff
@@ -582,9 +581,8 @@ class Signup extends React.Component {
 							flowName={ this.props.flowName }
 						/>
 					) }
-				<TransitionGroup component="div" className="signup__steps">
```

Comment:
Why are we getting rid of TransitionGroup here?

Reply:
Hey @shaunandrews - what was the reasoning on this? 

I think it's because `TransitionGroup` does some absolute positioning so that we can't just have the previews underneath. That would, I assume apply to the `CSSTransition` below, too.

Reply:
Yes, its related to the positioning of the previews. There's likely some workarounds we could do to keep the transitions (as they exist in production) in place, but it felt like too much work for little benefit at this time.

Once we have a fixed-height area for each step, then it will be easier to bring the transitions back.